lib/codebreaker/classes/player.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Codebreaker
+  # player entity
+  class Player
+    attr_reader :name
+
+    extend Validator
+
+    def initialize(name)
+      @name = name
+    end
+
+    def self.valid?(name)
```

Comment:
Can you validate it in console with Codebreaker::Validator.validate_name? ?

Reply:
We can, but in this particular way we use validator exactly in our gem, like we use validator on back of our app. In console I will be catching all errors from gem.